src/client/components/modes/modes.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './modes.css';
+import ModesItem from '../modesItems/modesItem';
+import { Image3d, ImageHeadSet, ImageSpeaker, ImageExtra } from 'src/images';
+
+const Modes = () => {
+  const imageArray = [{
```

Comment:
Good, but I would suggest to move such data in a separate files, like constants or selectors or even use redux here, so you have all your data in one place